keps/sig-node/3085-pod-conditions-for-starting-completition-of-sandbox-creation/README.md

```diff
@@ -1307,6 +1350,14 @@ Describe the metrics themselves and the reasons why they weren't added (e.g., co
 implementation difficulties, etc.).
 -->
 
+New metrics may be added to the Kubelet status manager to surface fine grained
+information about updates to overall pod status as well as specific pod
+conditions.
```

Comment:
The current feature is introducing a new condition (managed by the Kubelet status manager) in pod status. The new metric mentioned above applies to all pod status updates (conditions, container status, etc) managed by the Kubelet status manager. So that is a broader change and not part of this feature.

Reply:
Addressed.